packages/node/lib/process-request.js

```diff
@@ -1,22 +1,75 @@
 const url = require('url');
-const removeProperties = require('lodash/omit');
-const removeOtherProperties = require('lodash/pick');
+const get = require('lodash/get');
+const set = require('lodash/set');
+const pick = require('lodash/pick');
+const merge = require('lodash/merge');
 const contentType = require('content-type');
 
 const objectToArray = require('./object-to-array');
 
+/**
+ * @param {Any} value the value to be redacted
+ * @returns A redacted string potentially containing the length of the original value, if it was a string
+ */
+function redactValue(value) {
+  return `[REDACTED${typeof value === 'string' ? ` ${value.length}` : ''}]`;
```

Comment:
Why include the length here?

Reply:
It was requested, here's a [slack convo](https://readmeio.slack.com/archives/C01GWHXF42E/p1624649395000700)

Reply:
Should there be a space between `REDACTED` and the length?